cryosparc2/protocols/protocol_cryosparc_3D_classification.py

```diff
@@ -379,14 +342,58 @@ def _updateClass(self, item):
         classId = item.getObjId()
         if classId in self._classesInfo:
             index, fn, row = self._classesInfo[classId]
-            fn += ":mrc"
+            fixVolume(fn)
             item.setAlignmentProj()
             vol = item.getRepresentative()
             vol.setLocation(index, fn)
             vol.setSamplingRate(calculateNewSamplingRate(vol.getDim(),
                                                          self._getInputParticles().getSamplingRate(),
                                                          self._getInputParticles().getDim()))
 
+    def _createModelFile(self, csOutputFolder, itera):
+        # Create model files for 3D classification
+        with open(self._getFileName('out_class'), 'w') as output_file:
+            output_file.write('\n')
```

Comment:
my suggestion would be to use emtable which is a star file reader/writer, but it's up to you of course.

Reply:
I use the pyem library to build the .star from the .cs files. The problem is that pyem builds them based on the path where cryosparc creates them. That's the reason why I have to parse those .star and correct them. I don't know if emtable can do things like that...